config.md

```diff
@@ -355,18 +355,18 @@ Runtime implementations MAY support any valid values for platform-specific field
 Hooks allow for the configuration of custom actions related to the [lifecycle](runtime.md#lifecycle) of the container.
 
 * **`hooks`** (object, OPTIONAL) MAY contain any of the following properties:
-  * **`prestart`** (array of objects, OPTIONAL) is an array of [pre-start hooks](#prestart).
-    Entries in the array contain the following properties:
-    * **`path`** (string, REQUIRED) with similar semantics to [IEEE Std 1003.1-2001 `execv`'s *path*][ieee-1003.1-2001-xsh-exec].
-      This specification extends the IEEE standard in that **`path`** MUST be absolute.
-    * **`args`** (array of strings, OPTIONAL) with the same semantics as [IEEE Std 1003.1-2001 `execv`'s *argv*][ieee-1003.1-2001-xsh-exec].
-    * **`env`** (array of strings, OPTIONAL) with the same semantics as [IEEE Std 1003.1-2001's `environ`][ieee-1003.1-2001-xbd-c8.1].
-    * **`timeout`** (int, OPTIONAL) is the number of seconds before aborting the hook.
-      If set, `timeout` MUST be greater than zero.
-  * **`poststart`** (array of objects, OPTIONAL) is an array of [post-start hooks](#poststart).
-    Entries in the array have the same schema as pre-start entries.
-  * **`poststop`** (array of objects, OPTIONAL) is an array of [post-stop hooks](#poststop).
-    Entries in the array have the same schema as pre-start entries.
+    * **`prestart`** (array of objects, OPTIONAL) is an array of [pre-start hooks](#prestart).
+        Entries in the array contain the following properties:
+        * **`path`** (string, REQUIRED) with similar semantics to [IEEE Std 1003.1-2001 `execv`'s *path*][ieee-1003.1-2001-xsh-exec].
+        This specification extends the IEEE standard in that **`path`** MUST be absolute.
+        * **`args`** (array of strings, OPTIONAL) with the same semantics as [IEEE Std 1003.1-2001 `execv`'s *argv*][ieee-1003.1-2001-xsh-exec].
+        * **`env`** (array of strings, OPTIONAL) with the same semantics as [IEEE Std 1003.1-2001's `environ`][ieee-1003.1-2001-xbd-c8.1].
+        * **`timeout`** (int, OPTIONAL) is the number of seconds before aborting the hook.
+        If set, `timeout` MUST be greater than zero.
+    * **`poststart`** (array of objects, OPTIONAL) is an array of [post-start hooks](#poststart).
+        Entries in the array have the same schema as pre-start entries.
```

Comment:
Oh hmm, either this has two too many or the others I've commented on actually need _four_ more spaces rather than simply two (which seems the more likely, given that pandoc is apparently picky about it being precisely "four spaces or a tab").